### ISSUE TITLE
installing pleroma in a Raspberry Pi

### ISSUE THREAD

**Issue by xaloc33** [YunoHost-Apps/pleroma_ynh#45]
Hi,

I have a yunohost server running on a raspberry Pi 3, I tried to install the pleroma app but it failed. I think that it fails because it tries to install the amd64 packages for esl-erlang and elixir.
I don't know if this is the only problem there will be with a different architecture or if this problem can be easily fixed, I have no knowledge on how packaging and app works.

Anyway thank you very much for the work that you have done and if it's a difficult problem to solve I might try to install the packages before hand and just remove the lines where it tries to install the amd64 packages.

Thank you again for the work and help,

Xaloc

**Comment by yalh76** [comment]
Hi @xaloc33, 

The actual failure is related to nodejs install on ARM: https://github.com/YunoHost/yunohost/pull/660

The PR is merged but I don't know when it will be available on Yunohost.

**Comment by xaloc33** [comment]
A ok, thank you very much for answering so quickly. I guess I'll just have to wait for the change to get to stable

**Comment by yalh76** [comment]
Will be in Yunohost 3.5: https://forum.yunohost.org/t/yunohost-3-5-testing-call-for-feedback

**Comment by yalh76** [comment]
Can you retry to install Pleroma ?

**Comment by xaloc33** [comment]
I've tried a bunch of times but has failed every time.

First time it couldn't get the erlang-solutions.asc. I just retried after checking that the url was ok and second time it worked.

Second time pleroma-ynh-deps failed because erlang-eldap was not being installed, I manually installed before trying again and then this bit worked.

Third time all dependencies seem to be installed, PostgreSQL database is created, nginx is configured, system user is configured, and while or after configuring php-fm I start getting a lot of warnings and finally fails saying:
'''
Warning: ** (Mix) Could not invoke task "ecto.migrate": 1 error found!
Warning: --force : Unknown option
'''
